ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-multi-ec-4.13.yaml

```diff
@@ -0,0 +1,111 @@
+base_images:
```

Comment:
We do not support cross arch upgrade, so no need to stat `multi` twice, to shorten the job names (as file name is part of the job name), also to keep the same name style as the other arch.